apps/web/src/services/asset-inventory/collector/modules/CollectorItemInfo.vue

```diff
@@ -145,16 +173,57 @@ const props = withDefaults(defineProps<Props>(), {
     type: '',
 });
 
+const collectorPageStore = useCollectorPageStore();
+const collectorPageState = collectorPageStore.$state;
+
 const state = reactive({
-    toggleActive: false,
-    toggleStatus: 'OFF',
+    schedule: computed(() => collectorPageState.schedules.find((schedule) => schedule.collector_info.collector_id === props.item.collectorId)),
+    collectorState: computed(() => state.schedule?.collector_info.state),
+    toggleStatus: computed(() => (state.collectorState === COLLECTOR_STATE.ENABLED ? 'ON' : 'OFF')),
+    nextSchedule: computed(() => {
+        if (state.schedule) {
+            const numbersArray = state.schedule.schedule.hours;
+            const hour = new Date().getHours();
+            const hasNextSchedule = numbersArray.find((num) => num > hour);
+
+            let closestValue = 0;
+
+            if (hasNextSchedule) {
+                closestValue = hasNextSchedule as number;
+            } else {
+                closestValue = 24 - Math.min(...numbersArray);
+            }
+
+            return closestValue;
+        }
+        return undefined;
+    }),
+    diffSchedule: computed(() => {
+        const today = new Date();
+        const year = today.getFullYear();
+        const month = today.getMonth() + 1;
+        const day = today.getDate();
+        let formatDate = '';
+
+        if (state.nextSchedule >= 24) {
+            formatDate = `${year}-${month}-${day + 1}`;
```

Comment:
nitpick: What happens if it's the last day? It would be advisable to safely handle it using `dayjs`.

Reply:
```typescript
const today = dayjs.utc();

if (state.nextSchedule >= 24) {
  formatDate = today.add(1, 'day').format('YYYY-MM-DD');
  ...
```